cms/djangoapps/contentstore/views/item.py

```diff
@@ -729,33 +729,33 @@ def _move_item(source_usage_key, target_parent_usage_key, user, target_index=Non
 
         if (valid_move_type.get(target_parent_type, '') != source_type and
                 target_parent_type not in parent_component_types):
-            error = 'You can not move {source_type} into {target_parent_type}.'.format(
+            error = _('You can not move {source_type} into {target_parent_type}.').format(
                 source_type=source_type,
                 target_parent_type=target_parent_type,
             )
         elif source_parent.location == target_parent.location:
-            error = 'You can not move an item into the same parent.'
+            error = _('You can not move an item into the same parent.')
         elif source_item.location == target_parent.location:
-            error = 'You can not move an item into itself.'
+            error = _('You can not move an item into itself.')
         elif is_source_item_in_target_parents(source_item, target_parent):
-            error = 'You can not move an item into it\'s child.'
+            error = _('You can not move an item into it\'s child.')
         elif target_parent_type == 'split_test':
-            error = 'You can not move an item directly into content experiment.'
+            error = _('You can not move an item directly into content experiment.')
         elif source_index is None:
-            error = '{source_usage_key} not found in {parent_usage_key}.'.format(
+            error = _('{source_usage_key} not found in {parent_usage_key}.').format(
                 source_usage_key=unicode(source_usage_key),
                 parent_usage_key=unicode(source_parent.location)
             )
         else:
             try:
                 target_index = int(target_index) if target_index is not None else None
                 if len(target_parent.children) < target_index:
-                    error = 'You can not move {source_usage_key} at an invalid index ({target_index}).'.format(
+                    error = _('You can not move {source_usage_key} at an invalid index ({target_index}).').format(
                         source_usage_key=unicode(source_usage_key),
                         target_index=target_index
                     )
             except ValueError:
-                error = 'You must provide target_index ({target_index}) as an integer.'.format(
+                error = _('You must provide target_index ({target_index}) as an integer.').format(
```

Comment:
@mushtaqak My only concern is that some of the above messages are not user friendly. We can release the changes now but we should create a new ticket to improve the log messages.